docs/index.asciidoc

```diff
@@ -401,7 +401,7 @@ A default of 4095 is set in internally.
 What mode do you want the file input to operate in. Tail a few files or
 read many content-complete files. Read mode now supports gzip file processing.
 
-If `read` is specified, these settings can be used:
+If `tail` is specified, these settings can be used:
```

Comment:
This isn't quite right - `file_completed_action` and `file_completed_log_path` are only relevant with `read` mode, but `ignore_older` works across both modes.

What this PR points out though, is that this section could do with some improvement in clarity as to which settings are used/ignored across the different modes. I've created an issue (#294) to track this